src/components/FormDatePicker.vue

```diff
@@ -28,7 +29,7 @@
 
   const uniqIdsMixin = createUniqIdsMixin();
 
-  const datetimeStdFormat = 'YYYY-MM-DDThh:mm:ssZZ';
+  const datetimeStdFormat = 'YYYY-MM-DDTHH:mm:ssZZ';
```

Comment:
The capital `H` formats the value in 24-hour time, preserving the difference between AM/PM, and allowing the modeler to correctly interpret the date (as the modeler expects a (24-hour) ISO format date string.